website/content/docs/sync/github.mdx

```diff
@@ -72,7 +96,7 @@ If you decide to go with an access token, here's how you can configure a sync de
 1. Create secrets you wish to sync with a target GitHub repository for Actions.
 
   ```shell-session
-  $ vault kv put -mount='my-kv' my-secret foo='bar'
+  $ vault kv put -mount='my-kv' my-secret foo='bar' baz='quux'
```

Comment:
It would helpful for readers if we used more meaningful values for the command parameters